benches/bench.rs

```diff
@@ -0,0 +1,144 @@
+use async_memcached::Client;
+use criterion::{criterion_group, criterion_main, Criterion};
+use tokio::runtime::Runtime;
+
+const LARGE_PAYLOAD_SIZE: usize = 1000 * 1024; // Memcached's ~default maximum payload size
+
+async fn setup_client() -> Client {
+    Client::new("tcp://127.0.0.1:11211")
+        .await
+        .expect("failed to create client")
+}
```

Comment:
We're just using the raw client to prove that disabling Nagle's algorithm helped? (i.e. this is not the benchmark that compares protobuf over UDS vs protobufs over mmap vs raw memcached?